### DISCUSSION TITLE
Does remote logging for scheduler/dag processor/gunicorn logs ?

### DISCUSSION THREAD

**Discussion by ginolegigot** [apache/airflow#29239] [Q&A]
Hello!
I was wondering, does remote logging also work for scheduler/dag processor/gunicorn logs ? In the GCS bucket I only dags/tasks logs, so i dont know if i did not properly configured it or if it is a normal behavior.
Thanks for your time!

**Comment by potiuk** [comment]
This is the default behaviour yes. If you want to also send remote logging for other logs, you can use https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/logging-tasks.html#advanced-configuration and modify LOGGING_CONFIG as you wish.

**Reply by ginolegigot** [reply]
Oki thanks!
I read the doc but do you have some examples to provide ?

**Reply by potiuk** [reply]
The only examples are in the docs. But you can likely google some more.
